instance/management/commands/instance_statistics_csv.py

```diff
@@ -276,6 +285,38 @@ def log_line(line):
             logger_=log_line,
         )
 
+    def collect_active_users(
+            self,
+            out,
+            domain_names,
+            start_date,
+            end_date
+    ):
+        """Generate the active user statistics CSV."""
+        instances = self.get_instances_from_domain_names(domain_names)
```

Comment:
@janimo, we already run a python script on each instance's active appserver(s) to collect some data. Can't the code to do the querying be added to that? That way, the calling code in Ocim need not know about what is being done and how to collect this data?

Reply:
I considered that possibility but it meant touching more parts of the whole setup and also looked harder to test. I'll look into that.